enterprise/audit/table.go

```diff
@@ -147,6 +146,15 @@ var AuditableResources = auditMap(map[any]map[string]Action{
 		"ip_address":       ActionIgnore,
 		"scope":            ActionIgnore,
 	},
+	// TODO: track an ID here when the below ticket is completed:
+	// https://github.com/coder/coder/pull/6012
+	&database.License{}: {
+		"id":          ActionIgnore,
+		"uploaded_at": ActionIgnore,
+		"jwt":         ActionIgnore,
+		"exp":         ActionTrack,
+		"uuid":        ActionIgnore,
```

Comment:
IMO it might be helpful to track the `uuid` and `uploaded_at`. Even though it won't change it's still nice to help identify the license.

Reply:
Good idea!

Reply:
Fixed!